src/components/views/settings/tabs/user/VoiceUserSettingsTab.tsx

```diff
@@ -183,22 +201,60 @@ export default class VoiceUserSettingsTab extends React.Component<{}, IState> {
         return (
             <div className="mx_SettingsTab mx_VoiceUserSettingsTab">
                 <div className="mx_SettingsTab_heading">{ _t("Voice & Video") }</div>
+                { requestButton }
                 <div className="mx_SettingsTab_section">
-                    { requestButton }
+                    <span className="mx_SettingsTab_subheading">{ _t("Voice settings") }</span>
                     { speakerDropdown }
                     { microphoneDropdown }
+                </div>
+                <div className="mx_SettingsTab_section">
+                    <span className="mx_SettingsTab_subheading">{ _t("Video settings") }</span>
                     { webcamDropdown }
                     <SettingsFlag name='VideoView.flipVideoHorizontally' level={SettingLevel.ACCOUNT} />
-                    <SettingsFlag
-                        name='webRtcAllowPeerToPeer'
-                        level={SettingLevel.DEVICE}
-                        onChange={this.changeWebRtcMethod}
-                    />
-                    <SettingsFlag
-                        name='fallbackICEServerAllowed'
-                        level={SettingLevel.DEVICE}
-                        onChange={this.changeFallbackICEServerAllowed}
-                    />
+                </div>
+
+                <div className="mx_SettingsTab_heading">{ _t("Advanced") }</div>
+                <div className="mx_SettingsTab_section">
+                    <span className="mx_SettingsTab_subheading">{ _t("Voice processing") }</span>
+                    <div className="mx_SettingsTab_section">
+                        <LabelledToggleSwitch
+                            value={this.state.audioAutoGainControl}
+                            onChange={(v) => {
+                                MediaDeviceHandler.instance.setAudioAutoGainControl(v);
+                                this.refreshAudioSettings();
```

Comment:
I am not sure if there is much benefit to having `refreshAudioSettings()` - we could just call `setState` directly; though I guess setting the settings could potentially fail, so let's leave it this way but let's `await MediaDeviceHandler.instance.setAudioAutoGainControl(v);`

Reply:
Done, thanks for pointing it out.

(I guess `SettingsStore.watchSetting()` could also make this cleaner.)